### ISSUE TITLE
Offline mode doesn't work

### ISSUE THREAD

**Issue by safonik** [MyEtherWallet/MyEtherWallet#4815]
###### Info

- **Version**: 6.9.3, 6.9.2, 6.9.1, 6.9.0-hotfix.1, 6.9.0, 6.8.1-hotfix.1.
- **Browser**: Chrome, Firefox
- **OS**: Any

###### Steps to reproduce

1. Open index.html
2. Get just white screen, the page does not open because the fonts that are imported in CSS cannot be loaded (
https://fonts.googleapis.com/css2?family=DM+Sans:ital,opsz,wght@0,9..40,200;0,9..40,300;0,9..40,400;0,9..40,500;0,9..40,600;0,9..40,700;0,9..40,800;0,9..40,900;1,9..40,400;1,9..40,500;1,9..40,600&display=swap)
![offline-error](https://github.com/MyEtherWallet/MyEtherWallet/assets/34992277/f285dae1-a63e-45c5-b0a0-fa1b7a4d63cc)

###### What do you expect to happen?
Offline bundle should work correctly in a completely offline environment

**Comment by ghthor** [comment]
Also getting this error with offline mode in 6.8.1 & 6.8.1-hotfix.1 after page loads without error and I try to select to open a software wallet.

```
TypeError: this.$amplitude is undefined
    trackAccessWalletAmplitude handlerAnalytics.mixin.js:158
    mounted TheAccessWalletLayout.vue:365
```

![Peek 2024-01-20 10-01](https://github.com/MyEtherWallet/MyEtherWallet/assets/160298/e55e933f-6a12-495f-ae89-ceb78ba23492)

**Comment by safonik** [comment]
> this.$amplitude is undefined

It was fixed in version 6.9.0, but the problem with loading index.html in a completely offline environment remained

**Comment by safonik** [comment]
Version 6.9.2 still contains this bug

**Comment by safonik** [comment]
v. 6.9.3, bug is still here. It is very strange.

**Comment by JHGrove3** [comment]
Please fix this bug.
Is there an earlier version that will work in offline mode?

**Comment by gamalielhere** [comment]
@safonik thanks for the report, can you take a look at this PR and see if fixes the issue for you. Basically we were loading google fonts, I've now changed that to be on a local import instead. https://github.com/MyEtherWallet/MyEtherWallet/pull/4864

**Comment by gamalielhere** [comment]
Here's the build from PR #4864. you can also find it commented by our bot on the PR: https://www.mewbuilds.com/offline/OFFLINE-6b0c53af2e8b38b70491449fc1d385ac62676d62.zip

**Comment by safonik** [comment]
@gamalielhere PR https://github.com/MyEtherWallet/MyEtherWallet/pull/4864 fixes the issue 👍

**Comment by safonik** [comment]
Fixed in version 6.9.4
